lib/rack/utils.rb

```diff
@@ -360,19 +360,6 @@ def delete_cookie_header!(header, key, value = {})
     end
     module_function :delete_cookie_header!
 
-    # Return the bytesize of String; uses String#size under Ruby 1.8 and
-    # String#bytesize under 1.9.
-    if ''.respond_to?(:bytesize)
-      def bytesize(string)
-        string.bytesize
-      end
-    else
-      def bytesize(string)
-        string.size
-      end
-    end
-    module_function :bytesize
-
```

Comment:
For the sake of gracefully deprecating breaking changes to public API, suggest keeping this as
```ruby
def bytesize(string)
  warn 'Rack::Utils.bytesize is deprecated in Rack 2.0 and will be removed in Rack 2.1. Use String#bytesize instead.'
  string.bytesize
end
```

Ref. https://github.com/rack/rack-contrib/pull/124 which used this public API.